src/SFA.DAS.ApplyService.InternalApi/Startup.cs

```diff
@@ -206,9 +204,8 @@ private void ConfigureDependencyInjection(IServiceCollection services)
 
             services.AddTransient<IRoatpApiClient, RoatpApiClient>();
             services.AddTransient<IRoatpTokenService, RoatpTokenService>();
-            services.AddTransient<IKeyProvider,PlaceholderKeyProvider>();
-            services.AddTransient<IStorageService,StorageService>();
-            services.AddTransient<IEncryptionService, EncryptionService>();
+            //services.AddTransient<IKeyProvider,PlaceholderKeyProvider>();
```

Comment:
Can these be removed?

Reply:
sure can, they are all about encryption and storage, which is now subcontracted to QnA